TROGDOR.md

```diff
@@ -123,6 +123,14 @@ ProcessStopFault stops a process by sending it a SIGSTOP signal.  When the fault
 ### NetworkPartitionFault
 NetworkPartitionFault sets up an artificial network partition between one or more sets of nodes.  Currently, this is implemented using iptables.  The iptables rules are set up on the outbound traffic from the affected nodes.  Therefore, the affected nodes should still be reachable from outside the cluster.
 
+External Processes
+========================================
+Trogdor supports running external processes. This is a generic way to run any process in the system via a configurable command - be it a Python program, bash script, docker image, etc.
+
+### ExternalCommandWorker
+ExternalCommandWorker starts an external command defined by the ExternalCommandSpec. It essentially allows you to run any command on any Trogdor agent node and give it any arbitrary JSON input.
```

Comment:
Maybe "ExternalCommandWorker starts an process to execute an external command....", also remove "and give it any arbitary JSON input".

Then in next paragraph where it discusses the communication:

+ The worker communicates with the external process via its stdin, stdout and stderr in a JSON  protocol.